shared/utils/queue.go

```
@@ -1,46 +1,279 @@
+						slog.Debug("No message found in any queue, sleeping")
+						// Sleep briefly to avoid CPU spinning when queues are empty
+						time.Sleep(1 * time.Second)
+						return
+					}
+					slog.Debug("Found message", "subject", msg.Subject, "priority", priority)
+
+					// Get the UUID from the ticket
+					msg_id, err := uuid.FromBytes(msg.Data())
+					if err != nil {
+						slog.Error("Failed to parse message ID", "error", err, "data", string(msg.Data()))
+						msg.Nak() // Negative acknowledgment, message will be redelivered
```

golangci-lint (1.64.8)

Error return value of `msg.Nak` is not checked

(errcheck)